sections/quick-order-form.liquid

```diff
@@ -105,20 +126,57 @@
             </thead>
 
             <tbody>
-              {%-
-                render 'quick-order-product-row',
-                item: collections['shop-all'].products[0],
-                image: collections['shop-all']products[0].featured_media,
-              -%}
-              {%- for variant in collections['shop-all'].products[0].variants -%}
-                {%-
-                  render 'quick-order-list-row',
-                  item: variant,
-                  image: variant.image,
-                  sku: variant.sku,
-                  variant: variant,
-                -%}
-              {%- endfor -%}
+              {% if search.performed %}
+                {% paginate search.results by 5 %}
+                  {% for product in search.results %}
+                    {%-
+                      render 'quick-order-product-row',
+                      item: product,
+                      image: product.featured_media,
+                    -%}
+                    {%- for variant in product.variants -%}
+                      {% if variant.matched %}
+                        {%-
+                          render 'quick-order-list-row',
+                          item: variant,
+                          image: variant.image,
+                          sku: variant.sku,
+                          variant: variant,
+                        -%}
+                      {% endif %}
+                    {% endfor %}
+                  {% endfor %}
+                  <tr>
+                    <td>
+                      {% render 'quick-order-pagination', paginate: paginate, anchor: '' %}
+                    </td>
+                  </tr>
+                {% endpaginate %}
+              {% else %}
+                {% paginate collections.all.products by 5 %}
```

Comment:
Might make sense to put `5` in a local variable, just to make changing it easier for all the `paginate`s